docs/scenarios_authoring/BasicNodes.md

```diff
@@ -11,9 +11,9 @@ Most of the nodes, with source and sink nodes being notable exceptions, have inp
 Sinks and filters can be disabled by selecting `Disable` checkbox. 
 
 
-## Variable component
+## Variable 
 
-A Variable component is used to declare a new variable; in the simplest form a variable declaration looks like in the example  below. As the event was read from the Kafka topic, the `#input` variable stores its content and  its value is assigned to a newly declared `myFirstVariable` variable. 
+A Variable node is used to declare a new variable; in the simplest form a variable declaration looks like in the example  below. As the event was read from the Kafka topic, the `#input` variable stores its content and  its value is assigned to a newly declared `myFirstVariable` variable. 
```

Comment:
You're right that we've inconsistency here but I'm afraid it should be component here and that page should be called Basic Components not Basic Nodes, component is used deliberately here to be in line with https://nussknacker.io/documentation/about/GLOSSARY